junitparser/cli.py

```diff
@@ -56,6 +58,11 @@ def _parser(prog_name=None):  # pragma: no cover
     merge_parser.add_argument(
         "output", help='Merged XML Path, setting to "-" will output console'
     )
+    merge_parser.add_argument(
+        "--suite-name",
+        help="Name added to <testsuites>.",
+        default="",
```

Comment:
Why a default of empty string when above `result.name` is not set with the default value. There should be no default turning `args.suite_name` to `None`.

With `if suite_name:` (which tests for not None and not empty string), the resulting behaviour is the same, meaning there is no suite name if `--suite-name` is not set or set to an empty string.

Reply:
Good point, pushed a change for it